crates/proto/proto/zed.proto

```diff
@@ -56,12 +56,12 @@ message Envelope {
         UpdateProjectCollaborator update_project_collaborator = 30;
         RemoveProjectCollaborator remove_project_collaborator = 31;
 
-        GetDefinition get_definition = 32;
```

Comment:
Not that part, please.
While we own the stack, in theory, multiple different versions may exist in the same remote, shared project and we have to be more rigorous when it comes to the protocol.

Reply:
Hmm, I might have a bad knowledge on that, actually.

We have a CI lint for the incompatible changes there, so let's first fix the CI issues and see if any migration-related arise.

Reply:
Looks like the CI explicitly blocks it at least (https://github.com/zed-industries/zed/actions/runs/16005673955/job/45152981046) therefore reverted the protobuf related changes.